### PR TITLE
xds: add skipNetCmd flag to fallback test client

### PR DESCRIPTION
**Pull request by mohanli-ml**
We are setting up fallback test based on TD. Currently the test client is compiled in google3, so we must run it in a container so that the client can have the GRTE dependency. However, container does not have `ip`, `iptables`, etc network command, so we plan to run the network command outside of the container. To do this, add a new flag `skipNetCmd` to skip network commands inside the test client.

### CODE REVIEW

File: interop-testing/src/main/java/io/grpc/testing/integration/GrpclbFallbackTestClient.java [hunk 1]
```diff
@@ -32,35 +32,32 @@
...
  */
 public final class GrpclbFallbackTestClient {
-  private static final Logger logger =
-      Logger.getLogger(GrpclbFallbackTestClient.class.getName());
+  private static final Logger logger = Logger.getLogger(GrpclbFallbackTestClient.class.getName());
 
-  /**
-   * Entry point.
-   */
+  /** Entry point. */
```
Comment (dapengzhang0) [inline interop-testing/src/main/java/io/grpc/testing/integration/GrpclbFallbackTestClient.java:43]: nit: Can you please revert unrelated changes to make it easier to review?

File: interop-testing/src/main/java/io/grpc/testing/integration/GrpclbFallbackTestClient.java [hunk 2]
```diff
@@ -172,29 +177,30 @@ private void tearDown() {
     }
   }
 
-  private static void runShellCmd(String cmd) throws Exception {
+  private static void runShellCmd(String cmd, boolean skipNetCmd) throws Exception {
```
Comment (dapengzhang0) [inline interop-testing/src/main/java/io/grpc/testing/integration/GrpclbFallbackTestClient.java:180]: I would suggest make the method non-static, so you don't have to add a boolean argument.

**Comment by dapengzhang0** [comment]
LGTM in general. Just two minor comments.

File: interop-testing/src/main/java/io/grpc/testing/integration/GrpclbFallbackTestClient.java (hunk 1 above)
Comment (mohanli-ml) [reply interop-testing/src/main/java/io/grpc/testing/integration/GrpclbFallbackTestClient.java:43]: The style change is because I used `google-java-format-1.9-all-deps.jar` to format the code. Maybe let's keep the change as it obeys the style guidance?

File: interop-testing/src/main/java/io/grpc/testing/integration/GrpclbFallbackTestClient.java (hunk 2 above)
Comment (mohanli-ml) [reply interop-testing/src/main/java/io/grpc/testing/integration/GrpclbFallbackTestClient.java:180]: Done

**Review by dapengzhang0 — APPROVED** [review]
